hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/TransactionDispatcher.java

```diff
@@ -124,12 +132,55 @@ public Fees dispatchComputeFees(@NonNull final FeeContext feeContext) {
 
         try {
             final var handler = getHandler(feeContext.body());
-            return handler.calculateFees(feeContext);
+            var fee = handler.calculateFees(feeContext);
+            return scale(feeContext.body(), fee);
         } catch (UnsupportedOperationException ex) {
             throw new HandleException(ResponseCodeEnum.INVALID_TRANSACTION_BODY);
         }
     }
 
+    // Scaling the fees if certain conditions are met.
+    // E.g. when we do a ScheduleCreate we want to scale the fees based on what transaction type is scheduled
+    private static Fees scale(TransactionBody transactionBody, Fees fees) {
+        int multiplier = 1; // by default, we are not scaling
+
+        final var functionality = transactionBody.data().kind();
+        // We are only scaling the ScheduleCreate. In the future we can add more functionalities.
+        switch (functionality) {
+            case SCHEDULE_CREATE -> multiplier = getScheduleMultiplier(transactionBody);
```

Comment:
This is the wrong concept. We do not want transaction-specific code in the Hedera app if we can avoid it.

Why is the multiplier applied here? Is this a generic concept that will be used for multiple transaction types? I would have expected this to happen in `ScheduleCreateHandler.calculateFees()`

Reply:
`Is this a generic concept that will be used for multiple transaction types?`
Yes, I though that we will have scaling for other types as well, but I might be wrong. At the beginning it was implemented inside `ScheduleCreateHandler.calculateFees()` but then I moved it here.

Let see if this PR is needed at all and then I will resolve your comments 👍